File: app/src/main/java/com/stedi/randomimagegenerator/app/view/components/ListSpaceDecoration.java
package com.stedi.randomimagegenerator.app.view.components;

import android.graphics.Rect;
import android.support.v7.widget.RecyclerView;
import android.view.View;

public class ListSpaceDecoration extends RecyclerView.ItemDecoration {
    private final int vSpace;
    private final int lrSpace;

    public ListSpaceDecoration(int vSpace, int lrSpace) {
        this.vSpace = vSpace;
        this.lrSpace = lrSpace;
    }

    @Override
    public void getItemOffsets(Rect outRect, View view, RecyclerView parent, RecyclerView.State state) {
        if (parent.getChildAdapterPosition(view) == 0)
            outRect.top = vSpace;
        outRect.bottom = vSpace;
        outRect.left = lrSpace;
        outRect.right = lrSpace;
    }
}

File: app/src/main/java/com/stedi/randomimagegenerator/app/view/components/ListSpaceDecoration.kt
package com.stedi.randomimagegenerator.app.view.components

import android.graphics.Rect
import android.support.v7.widget.RecyclerView
import android.view.View

class ListSpaceDecoration(private val vSpace: Int, private val lrSpace: Int) : RecyclerView.ItemDecoration() {

    override fun getItemOffsets(outRect: Rect, view: View, parent: RecyclerView, state: RecyclerView.State) {
        if (parent.getChildAdapterPosition(view) == 0) {
            outRect.top = vSpace
        }
        outRect.bottom = vSpace
        outRect.left = lrSpace
        outRect.right = lrSpace
    }
}